config/config.go

```diff
@@ -0,0 +1,213 @@
+package config
+
+import (
+	"encoding/base64"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/Fresh-Tracks/bomb-squad/util"
+	"github.com/prometheus/common/model"
+	promcfg "github.com/prometheus/prometheus/config"
+	yaml "gopkg.in/yaml.v2"
+)
+
+type Configurator interface {
+	Read() []byte
+	Write([]byte) error
+	GetLocation() string
+}
+
+type BombSquadLabelConfig map[string]string
+
+type BombSquadConfig struct {
+	SuppressedMetrics map[string]BombSquadLabelConfig
+}
+
+func ReadBombSquadConfig(c Configurator) BombSquadConfig {
+	b := c.Read()
+	bscfg := BombSquadConfig{}
+	err := yaml.Unmarshal(b, &bscfg)
+	if err != nil {
+		log.Fatalf("Couldn't unmarshal into config.BombSquadConfig: %s\n", err)
```

Comment:
Typically it is better to return an error here. If this is truly exceptional consider a panic instead. Looks like there are several Fatalf calls in this file